flashlight/pkg/speech/criterion/AutoSegmentationCriterion.h

```diff
@@ -91,8 +91,8 @@ class AutoSegmentationCriterion : public SequenceCriterion {
 };
 
 using ASGLoss = AutoSegmentationCriterion;
-} // namespace asr
-} // namespace app
+} // namespace speech
+} // namespace pkg
 } // namespace fl
 
-CEREAL_REGISTER_TYPE(fl::app::asr::AutoSegmentationCriterion)
+CEREAL_REGISTER_TYPE(fl::pkg::speech::AutoSegmentationCriterion)
```

Comment:
💥 

cc @tlikhomanenko, this will break old ASG models from lexfree/other stuff potentially